packages/hathor-rpc-handler/src/rpcMethods/sendTransaction.ts

```diff
@@ -0,0 +1,188 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { z } from 'zod';
+import type { HathorWallet } from '@hathor/wallet-lib';
+import type { OutputSendTransaction } from '@hathor/wallet-lib/lib/wallet/types';
+import {
+  TriggerTypes,
+  PinConfirmationPrompt,
+  PinRequestResponse,
+  TriggerHandler,
+  RequestMetadata,
+  SendTransactionRpcRequest,
+  SendTransactionConfirmationPrompt,
+  SendTransactionConfirmationResponse,
+  SendTransactionLoadingTrigger,
+  SendTransactionLoadingFinishedTrigger,
+  RpcResponseTypes,
+  RpcResponse,
+  RpcMethods,
+} from '../types';
+import {
+  PromptRejectedError,
+  InvalidParamsError,
+  SendTransactionError,
+  InsufficientFundsError,
+} from '../errors';
+import { validateNetwork } from '../helpers';
+
+const sendTransactionSchema = z.object({
+  method: z.literal(RpcMethods.SendTransaction),
+  params: z.object({
+    network: z.string().min(1),
+    outputs: z.array(z.object({
+      address: z.string().optional(),
+      value: z.string().transform(val => BigInt(val)),
```

Comment:
Isn't it optional when it's a data output?

Reply:
It is, changed it to optional and added a better refine method to make it required only when not data output